CODE/explain_human_resps.py

```diff
@@ -47,8 +47,6 @@ def parse_args(argv):
     parser.add_argument(
         '--enable_permutation_shap', action='store_true',
         help="Enable SHAP for permutation runs. Permutations will default to not calculating SHAP values\n(using this flag will make permutations training significantly longer.")
```

Comment:
If I understood correctly, you would like:
1. Default to calculating SHAP values for regular (non-permutation) runs
2. Default to not calculating SHAP values for permutation runs

Replacing `--disable_shap` with `--enable_permutation_shap` will not allow us to disable calculating SHAP values in regular runs. Perhaps default to not calculating SHAP values for both regular and permutation runs and change flag to `--enable_shap`:
```suggestion
        '--enable_shap', action='store_true',
        help="Enable calculation of SHAP values. Runs will default to not calculating SHAP values\n(using this flag will make training significantly longer.")
```

Reply:
My inclination would be disabling shap when `-permutations` is enabled as it doesn't make sense to analyze shap for random permutations.